src/player.h

```diff
@@ -5,6 +5,7 @@
 #include "item.h"
 #include "player_activity.h"
 #include "weighted_list.h"
+#include "morale.h"
```

Comment:
It must have been included indirectly, there's a vector of type morale_point in player, which AFAIK requires a full definition of the type, not just a declaration.